README.md

```diff
@@ -64,7 +64,17 @@ Landrush assigns your vm's hostname from either the vagrant config (see the `exa
 
 Every time a VM is started, its IP address is automatically detected and a DNS record is created that maps the hostname to its IP.
 
-If for any reason the auto-detection detects no IP address or the wrong IP address, or you want to override it, you can do like so:
+If for any reason the auto-detection detects no IP address or the wrong IP address you can control how it looks for it.
+
+You can start by excluding interfaces matching an array of regex patterns (the value shown here is the default):
+
+    config.landrush.exclude = [/lo[0-9]*/, /docker[0-9]+/, /tun[0-9]+/]
```

Comment:
I find _config.landrush.exclude_ a bit too generic and non telling. To align with _config.landrush.host_ip_address_, _config.landrush.host_interface_excludes_ might be better.

Personally I find _host_ip_address_ already not too good, since for me we are actually talking about the VM/guest IP. Or better the host visible guest IP. To me _host_ip_address_ is the IP of my machine on which I am running the VM. But changing this at this stage might be not be so good. WDYT?